app/uk/gov/hmrc/tai/util/DateHelper.scala

```diff
@@ -43,4 +45,12 @@ object DateHelper {
     monthAndYearRegex.findFirstIn(date).getOrElse("")
   }
 
+  def dateTimeFormat(dt: DateTime): String = {
+    val month = dt.monthOfYear.getAsText
+    val day = dt.getDayOfMonth
+    val year = dt.getYear
+    val time = dt.toLocalTime.toString("HH:mm")
+    s"$day $month $year at $time"
```

Comment:
would something like this ```DateTimeFormatter.ofPattern("dd MMM yyyy 'at' HH:mm")``` work?

Reply:
i see it's joda time in that case: 
```def dateTimeFormat(dt: DateTime): String = dt.toLocalTime.toString(DateTimeFormat.forPattern("dd MMM yyyy 'at' HH:mm"))```
also add a test with some DateTime that checks it follows the string pattern

Reply:
https://stackoverflow.com/questions/63737832/datetime-format-needed-chnages-as-expected